sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts

```diff
@@ -98,6 +99,20 @@ export const validateMptPAT = (
     if (result!.accountId !== claims!.aid) {
       validationFailureCallback(ServiceErrorMessageConstants.WORKSPACE_MISMATCH_ERROR);
     }
+    const currentTime = Date.now();
```

Comment:
you can move this validate logic in a function to keep code modular.

Reply:
something like this 
    if (isTokenExpiringSoon(expirationTime, currentTime) && shouldWarnAboutExpiry()) {
      warnAboutTokenExpiry(expirationTime, currentTime);
    }